modules/develop/pages/data-transforms/data-transforms.adoc

```diff
@@ -0,0 +1,231 @@
+= Run Data Transforms
+:description: Learn how to build and deploy data transforms in Redpanda.
+:page-aliases: reference:rpk/rpk-wasm/rpk-wasm.adoc, reference:rpk/rpk-wasm.adoc, reference:rpk/rpk-wasm/rpk-wasm-deploy.adoc, reference:rpk/rpk-wasm/rpk-wasm-generate.adoc, reference:rpk/rpk-wasm/rpk-wasm-remove.adoc, data-management:data-transform.adoc
+:latest-data-transforms-version: 0.0.0-20230830git604fcce
+
+Data transforms let you run common data streaming tasks, like filtering, scrubbing, and transcoding, within the Redpanda broker. For example, you may have consumers that require you to remove credit card numbers or convert JSON to Avro. Data transforms can also interact with the Redpanda Schema Registry to work with encoded data types. You can develop and manage data transforms with `rpk transform` commands.
+
+For more detailed information, see xref:./how-transforms-work.adoc[].
+
+== Limitations
```

Comment:
It feel sort of like a downer to list these up front. Should they go towards the end?